.github/workflows/application-signals-e2e-test.yml

```diff
@@ -38,53 +38,180 @@ jobs:
       - name: Upload main-build adot.jar to s3
         run: aws s3 cp ./aws-opentelemetry-agent-*-SNAPSHOT.jar s3://adot-main-build-staging-jar/aws-opentelemetry-agent.jar
 
-  java-ec2-default-e2e-test:
+  #
+  # PACKAGED DISTRIBUTION LANGUAGE VERSION COVERAGE
+  # DEFAULT SETTING: {Java Version}, EC2, AMD64, AL2
+  #
+
+  ec2-default-v8-amd64:
```

Comment:
nit: Can change all `ec2-default` references to `ec2` and all `ec2-asg` references to `asg` for simplicity

Reply:
Done